tests/contrib/test_sqlalchemy.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import unittest
+
+from sqlalchemy import Column, create_engine, Integer
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from oauth2client import GOOGLE_TOKEN_URI
+from oauth2client.client import OAuth2Credentials
+from oauth2client.contrib.sqlalchemy import CredentialsType, Storage
+
+Base = declarative_base()
+
+
+class DummyModel(Base):
+    __tablename__ = 'dummy'
+
+    id = Column(Integer, primary_key=True)
+    key = Column(Integer)  # we will query against this, because of ROWID
+    credentials = Column(CredentialsType)
+
+
+class TestSQLAlchemyStorage(unittest.TestCase):
+    engine = create_engine('sqlite://')
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
What's so expensive about this that it needs to be done in a `setUpClass` and could not be done in a `setUp`?